grafana/provisioning/dashboards/perf.json

```diff
@@ -435,7 +435,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "go_memstats_heap_objects{pod=\"$node\"}",
+          "expr": "go_memstats_heap_objects{namespace=\"$namespace\",pod=\"$node\"}",
```

Comment:
Testing this our on our cluster, and our metrics have `kubernetes_namespace` rather than `namespace` - versioning difference perhaps?

Reply:
Not really sure about this. We're using prometheus operator to deploy prometheus and grafana and in all of the templates installed by that the variable `namespace` is used. I can't see any examples of `kubernetes_namespace` being used.Tried searching the web to understand the difference by wasn't able to find anything relevant.

Reply:
Did some testing on this. It doesn't matter what the variable is called, as long as the query behind it is `label_values(namespace)` as per https://github.com/lightninglabs/lndmon/pull/60/files#diff-c1ba9705a49adb4580928d95e75b7973c367a5b1f5142a44ac5a9f97c3fccd1cR427

Reply:
@carlaKC just FYI, I was able to solve this with a `metric_relabel_configs` setting in Prometheus, copying the kubernetes labels into ones named `namespace` and `pod`.
@mrfelton: we're using the BKPR stack which has different naming schemes for the prometheus labels apparently. But I don't think that should be the concern for this PR if we can solve it with a label rewrite.

Reply:
sgtm, but I think we should add some docs to the PR noting the labeling in k8s and relabelling so that folks know how to use these additions?